db/migrate/20170421193150_add_embedded_methods_to_miq_ae_method.rb

```diff
@@ -0,0 +1,5 @@
+class AddEmbeddedMethodsToMiqAeMethod < ActiveRecord::Migration[5.0]
+  def change
+    add_column :miq_ae_methods, :embedded_methods, :string, array: true, default: []
```

Comment:
I think you want `default_value_for` in the model, not `:default` in the migration.